### ISSUE TITLE
Name is not returned in Creatable 'select-option' action

### ISSUE THREAD

**Issue by kino90** [JedWatson/react-select#3330]
I'm using both `Creatable` and normal `Select` elements in my app, but when I input a "new" value to a Creatable one, the "actionMeta" object has only the `action: 'select-option'` and not the name of the select field that triggered the action. I think they should behave in the same way.

I've made a sandbox here:
https://codesandbox.io/s/nw1pklvykj

If you choose an existing option (from both of the select) it outputs on the console: 
(notice: ' name: "standard" and name: "creatable")

```
Object {value: "ocean", label: "Ocean", color: "#00B8D9"}
Object {action: "select-option", option: undefined, name: "standard"}
Object {value: "ocean", label: "Ocean", color: "#00B8D9"}
Object {action: "select-option", option: undefined, name: "creatable"}
```

But if you input a "new value" on the Creatable it outputs : 
(notice: no name here!)

```
Object {label: "DarkGreen", value: "DarkGreen", __isNew__: true}
Object {action: "create-option"}
```

**Comment by TallOrderDev** [comment]
Running into this same issue. Would love to see consistency with the Meta object.

**Comment by chaitanyapilaka** [comment]
Curious to know if any work was done on this yet?

**Comment by sahilsharmafrank** [comment]
Any pointers or reference if this is tracked in an different ticket or an alternate solution to this

**Comment by amruthsrepo** [comment]
@sahilsharmafrank, how do I use that file in my react project?

**Comment by arjitaM** [comment]
Facing the same issue.

**Comment by ebonow** [comment]
Greetings,

It would seem that this bug was patched in v3.0.8 and this should no longer be issue.
<img width="791" alt="Screen Shot 2020-12-03 at 11 08 39 PM" src="https://user-images.githubusercontent.com/1007579/101314754-d3e35f00-381e-11eb-839c-94f29c426533.png">
